tests/e2e/Services/Databases/DatabasesBase.php

```diff
@@ -1951,6 +2072,18 @@ public function testInvalidDocumentStructure()
             'write' => ['user:' . $this->getUser()['$id']],
         ]);
 
+        $badTime = $this->client->call(Client::METHOD_POST, '/databases/' . $databaseId . '/collections/' . $collectionId . '/documents', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'documentId' => 'unique()',
+            'data' => [
+                'birthDay' => '2020-10-10 27:30:10+01:00',
+            ],
+            'read' => ['user:' . $this->getUser()['$id']],
+            'write' => ['user:' . $this->getUser()['$id']],
```

Comment:
FYI for other reviewers: this is handled by a downstream [PR](https://github.com/appwrite/appwrite/pull/3700).